src/web/data/userMapData.json

```diff
@@ -1047,12 +1076,14 @@
       "La Cote International School",
       "Medtronic",
       "Pädagogische Hochschule St. Gallen",
+      "SHMS",
```

Comment:
and here

Reply:
The issue mentioned just the abbreviations in these 3 cases.

Reply:
There are many abbreviations like these in the current file, so I thought that just the exact name mentioned in the issue has to be added to the list.

Reply:
It is better to use full forms. You can search for their full forms. Follow #9190

Reply:
I can do that if but can we make it a new d.FirstTimers issue to update all abbreviations as Full Forms? That might be more useful as it the new issue will require changing all the abbreviations in the file and not just these 3.

Reply:
@Ayush517, @RonakLakhotia was asking you to use full forms only for entries you added. Existing entries are not the concern of this PR.

> > Have you run `sort-usermap.js` to sort the entry?
> 
> No. I've done by manually.
> Checked all the entries twice.

You still need to run that script for two reasons: (1) the sorting logic may be different from what you had in mind, and (2) the sorting script is immune to human errors.

Reply:
1.) I'll update the abbreviations for UCA and SHMS, there's no full form for ALX
2.) I'll run the script on the changed file.
I'll have to close this pull request?

Reply:
No need to close this pull request, update the changes in your local branch and push them